docs/en/02_Developer_Guides/14_Files/06_Allowed_file_types.md

```diff
@@ -90,3 +90,43 @@ SilverStripe\MimeValidator\MimeUploadValidator:
       - 'text/plain'
       - 'text/calendar'
 ```
+
+## Adding new image types {#add-image-format}
+
+Silverstripe CMS support JPEG, GIF, PNG and WebP images format out of the box. Silverstripe CMS can be configured to support other less common images format (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image format.
```

Comment:
```suggestion
Silverstripe CMS support JPEG, GIF, PNG and WebP image formats out of the box and can be configured to support other less common image formats (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image formats.
```

Reply:
```suggestion
Silverstripe CMS support JPEG, GIF, PNG and WebP image formats out of the box. Silverstripe CMS can be configured to support other less common image formats (e.g.: AVIF). For this to work, your version of PHP and of the [`intervention/image` library](https://intervention.io/) must support these alternative image formats.
```

Reply:
Starting two sentences in a row with "Silverstripe CMS" reads a bit weird to me but it's not enough of an issue for me to push it. I'm fine with your suggested change.